### PR TITLE
Explode Creeper Effect and Ignition Condition

### PR DESCRIPTION
**Pull request by APickledWalrus**
### Description
This PR adds an effect to start/stop the ignition process of a creeper. I also added a condition to check if a creeper is in the ignition process, which requires Paper 1.13+.

---
**Target Minecraft Versions:**     Paper 1.13+ for the full effect and condition OR Spigot 1.14+ for just partial effect functionality
**Requirements:**     Paper for the condition, and for the ability to stop the ignition process.
**Related Issues:**     None

### CODE REVIEW

File: src/main/java/ch/njol/skript/conditions/CondIgnitionProcess.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+@Name("Ignition Process")
+@Description("Checks if a creeper is going to explode. Requires Paper 1.13+")
+@Examples({"if the last spawned creeper is going to explode:",
+			"\tloop all players in radius 3 of the last spawned creeper",
```
Comment (bensku) [inline src/main/java/ch/njol/skript/conditions/CondIgnitionProcess.java]: Use proper annotations to indicate requirements.

File: src/main/java/ch/njol/skript/effects/EffExplodeCreeper.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+
+@Name("Explode Creeper")
+@Description({"Starts the explosion process of a creeper or instantly explodes it.",
+				"This effect requires Paper 1.13+ or Spigot 1.14+.",
```
Comment (bensku) [inline src/main/java/ch/njol/skript/effects/EffExplodeCreeper.java]: Same here; there is an annotation for syntax requirements.

**Comment by APickledWalrus** [comment]
Should be okay now

**Comment by Nicofisi** [comment]
Correct me if I'm wrong, but couldn't the condition simply be a [PropertyCondition](https://github.com/SkriptLang/Skript/blob/21a52e247f9c51521b4c49f7f4874c9a1e159c41/src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java)?

**Comment by APickledWalrus** [comment]
I don’t think so, because of the word creeper in front of the condition. Ideally, it’d be nice to just have ``%livingentities% is/are ignited``, but that conflicts with the burning condition

**Comment by Nicofisi** [comment]
> I don’t think so, because of the word creeper in front of the condition.

If that's the only issue, you could still extend the PropertyCondition class to need less code, and only continue using your custom `Skript.registerCondition` instead of `PropertyCondition.register` for more customization :)

I'm nearly sure some class(es) already are like that.

**Comment by APickledWalrus** [comment]
That’s true, I will update this soon

**Comment by APickledWalrus** [comment]
Okay, I changed it to a property condition. It should be good now!

**Pull request merged**